app/views/pages/change_ECF_training_provider.html.erb

```diff
@@ -0,0 +1,112 @@
+
+<h2 class="govuk-heading-m">Change training provider for ECTs and mentors across 1 or 2 years</h2>    
+
+<h3 class="govuk-heading-s">If you’re changing provider for all ECTs and mentors starting training in the new academic year</h3>
+
+<p>Your school should contact your training providers first.</p> 
+
+<ol class="govuk-list govuk-list--number">
+  <li>Contact the new lead provider you want to work with.</li>
+  <li>Check that they have the capacity to train ECTs, and mentors where relevant, in the new academic year.</li>
+</ol>
+
+<p>Your school’s registered induction tutor is responsible for telling DfE how your school will deliver ECF-based training ahead of each academic year.</p>
+
+<p>We’ll email your school’s registered induction tutor in the summer term when it’s time to sign in to the Manage training for early career teachers service and set up training for the new year.</p>
+
+<p>They should then follow these steps.</p>
+
+<ol class="govuk-list govuk-list--number">
+  <li><a class="govuk-link" href="https://manage-training-for-early-career-teachers.education.gov.uk/users/sign_in">Sign in to the DfE service</a> and report that your school will use a different provider in the new academic year.</li>
+  <li>Register information about new ECTs and mentors who’ll start their training in the new academic year.</li>
+</ol>
+
+<p>They do not need to tell us which provider your school has changed to. Your lead provider will report the information to us directly once you set up training in the DfE service.</p>
+
+<p>We’ll confirm the changes with providers and update the DfE service as soon as possible. This may take a few weeks.</p>
+
+<h3 class="govuk-heading-s">If you’re changing providers for ECTs and mentors who have already started training</h3>
+
+<p>Your school should contact your training providers first.</p>
+
+<ol class="govuk-list govuk-list--number">
+  <li>Contact the new lead provider you want to work with. </li>
+  <li>Check that they have the capacity to train ECTs (and mentors where relevant) in years 1, 2 or both.</li>
+  <li>Confirm the date they can start delivering training for you.</li>
+  <li>Let your existing lead provider know about the change.</li>
+</ol>
+
+<p>Next, your school’s registered induction tutor must report the change to DfE. We need this information to pay the correct funding to each training provider.</p>
+
+<p class="govuk-body">Email <%= govuk_link_to "continuing-professional-development@digital.education.gov.uk", "mailto:continuing-professional-development@digital.education.gov.uk" %> and provide:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>your school’s name and unique reference number (URN)</li>
+  <li>the name of the new lead provider</li>
+  <li>the confirmed date the new lead provider will start training your ECTs (and their mentors if relevant)</li>
+  <li>the years you want to apply the change to (year 1, year 2, or both years)<li> 
+</ul>
```

Comment:
Given that we’ll (soon) have the support form to let schools contact us about this within the service, should we direct users there instead? 🤔

Reply:
I think we go live as is to support next week's comms, and then update once the forms are rolled out.